src/Image.php

```diff
@@ -609,7 +609,9 @@ protected function gc()
                 if (file_exists($file->getPathname()) && $file->getMTime() < $expire) {
                     // only deletes files ending with $this->suffix
                     if (substr($file->getFilename(), -($suffixLength)) == $this->suffix) {
+                        ErrorHandler::start();
                         unlink($file->getPathname());
+                        ErrorHandler::stop();
```

Comment:
Pretty sure that this needs to be `ErrorHandler::stop(true);` or you need to check the return value from `stop()` and do something with it. Thoughts @weierophinney?

Reply:
@akrabat, isn't this intentional, to just silence the warning?

Reply:
@akrabat `ErrorHandler::stop()` takes an optional boolean, `false` by default, to indicate whether or not it should _throw an exception_ if an error was caught. In this particular case, we're willing to ignore any warnings raised by `unlink()`, so no argument is necessary.